tests/BaGetter.Core.Tests/Support/Generator.cs

```diff
@@ -21,7 +21,7 @@ internal static Package GetPackage(string packageId, string version)
             //Use current date for each packages publish date, because later a date offset will be
             //calculated and leads to an overflow error of the offset because the default is year 0001.
             Published = DateTime.UtcNow,
-            Downloads = 1
+            Downloads = downloads ?? 1
```

Comment:
This seems redundant. You can use an `int` parameter (not nullable) with default value of `1`, then it can never be null and is automatically 1 when nothing is passed. Then just assign the parameter to `Downloads`.